src/Couchbase.Lite.Shared/API/Document/Document.cs

```diff
@@ -112,8 +112,14 @@ internal C4DocumentWrapper c4Doc
 
         internal virtual bool IsMutable => false;
 
+        /// <summary>
+        /// The RevisionID in Document class is a constant, while the RevisionID in MutableDocument class is not. Every time a MutableDocument is saved, 
+        /// its internal revision (via C4Document) will be updated. When a new MutableDocument is created, the document will not have C4Document 
+        /// nor Revision associated with it, therefore, the revisionID will be null.
+        /// The RevisionID format is opaque, which means it's format has no meaning and shouldn�t be parsed to get information.
+        /// </summary>
         [CanBeNull]
-        internal string RevID => c4Doc?.HasValue == true ? c4Doc.RawDoc->selectedRev.revID.CreateString() : null;
+        public string RevisionID => c4Doc?.HasValue == true ? c4Doc.RawDoc->selectedRev.revID.CreateString() : null;
```

Comment:
Leave out the stuff explaining about `C4Document`.  It is internal and the developer doesn't need to worry about it.  Just note that a newly created document will have a `null` RevisionID and that mutable documents will be updated on save.  Also, use `<see cref="MutableDocument" />` when you talk about mutable document so that it creates a clickable link in the API docs.